scripts/gen_coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+cmake -DCMAKE_BUILD_TYPE=Debug -GNinja -DCMAKE_BUILD_TYPE=Debug -DSRF_WITH_CODECOV=ON -DSRF_BUILD_PYTHON=ON -DSRF_BUILD_TESTS=ON -B ./build
+cmake --build ./build
+pip install -e ./build/python
+cd ./build && ctest && pytest ./build/python/tests
```

Comment:
Its it possible to generate coverage for libsrf and libpysrf separately?

Reply:
Yes; but its a bit of a pain, because you have to generate one set of gcda files with the c++ tests, compile them into a report, then delete all the files and repeat the process with the pytests.

Reply:
Does the report separate the two libraries?

Reply:
No, its basically like aggregated sampling. Even in the case of c++ tests there are multiple executables that need to be combined; we don't have a direct notion of which executable touched the given line, only that it was touched by one of tests that was run (and how many total times the line was touched).